micmac_li3ds/import_autocal.py

```diff
@@ -18,7 +16,7 @@ class ImportAutocal(Command):
     log = logging.getLogger(__name__)
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super(ImportAutocal, self).__init__(*args, **kwargs)
```

Comment:
Do we really need to bother with Python 2.x? I was thinking that we could just support Python 3.